### PR TITLE
Fat 1285 c11109

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-1285
***
https://foliotest.testrail.io/index.php?/cases/view/11109
***
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/963/allure/
***

### CODE REVIEW

File: cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      profileName: jobProfileForUpdateName,
+      acceptedType: NewJobProfile.acceptedDataType.marc
+    } }
+  ];
+
+  before('login', () => {
+    cy.getAdminToken()
+      .then(() => {
+        // InventorySearchAndFilter.getInstancesByIdentifierViaApi(oclcNumber.value)
+        //   .then(instances => {
```
Comment (MaksimTerekhov) [inline cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js:96]: remove comments pls

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 2]
```diff
@@ -278,13 +278,37 @@ export default {
...
 
-  fillCatalogedDate:() => {
-    cy.do(TextField('Cataloged date').fillIn(catalogedDate));
+  addSuppressFromDiscovery:() => {
+    const suppressFromDiscovery = 'profile.mappingDetails.mappingFields[0].booleanFieldAction';
+    const staffSuppress = 'profile.mappingDetails.mappingFields[1].booleanFieldAction';
+    const previouslyHeld = 'profile.mappingDetails.mappingFields[2].booleanFieldAction';
+    cy.do([
+      Select({ name: suppressFromDiscovery }).focus(),
+      Select({ name: suppressFromDiscovery }).choose('Mark for all affected records'),
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:287]: move to const

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 3]
```diff
@@ -278,13 +278,37 @@ export default {
...
+      Select({ name: staffSuppress }).choose('Mark for all affected records'),
+      Select({ name: previouslyHeld }).focus(),
+      Select({ name: previouslyHeld }).choose('Mark for all affected records')
+    ]);
+  },
+
+  addNatureOfContentTerms:() => {
+    const contentTerms = 'profile.mappingDetails.mappingFields[22].repeatableFieldAction';
+
+    cy.do(Select({ name: contentTerms }).focus());
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:298]: move to const

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+      JobProfiles.runImportFile();
+      JobProfiles.waitFileIsImported(nameMarcFileForCreate);
+      Logs.checkStatusOfJobProfile('Completed');
+      Logs.openFileDetails(nameMarcFileForCreate);
+      [FileDetails.columnName.srsMarc,
+        FileDetails.columnName.instance,
+      ].forEach(columnName => {
+        FileDetails.checkStatusInColumn(FileDetails.status.created, columnName);
+      });
+      FileDetails.checkSrsRecordQuantityInSummaryTable('1');
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js:159]: move value 1 to the variable

Comment (penfriend) [reply cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js:159]: in all methods

File: cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+          cy.visit(TopMenu.dataImportPath);
+          DataImport.uploadFile('marcFileForC11109.mrc', nameMarcFileForUpdate);
+          JobProfiles.searchJobProfileForImport(jobProfileForUpdateName);
+          JobProfiles.runImportFile();
+          JobProfiles.waitFileIsImported(nameMarcFileForUpdate);
+          Logs.checkStatusOfJobProfile('Completed');
+          Logs.openFileDetails(nameMarcFileForUpdate);
+          FileDetails.checkStatusInColumn(FileDetails.status.created, FileDetails.columnName.srsMarc);
+          FileDetails.checkStatusInColumn(FileDetails.status.updated, FileDetails.columnName.instance);
+          FileDetails.checkSrsRecordQuantityInSummaryTable('1');
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js:212]: the same here

File: cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js:96]: Done

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:287]: Unfortunately I can't move const to const

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:298]: Unfortunately I can't move const to const

File: cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js]: Done

File: cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/match-on-oclc-and-update-instance.cy.js]: Done

**Pull request merged**